.gitignore

```
@@ -1,4 +1,5 @@
 .cache/
 .config/
```

Zie latere comments over default paden
```suggestion
cache/
config/
```